jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2Test.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.maven.skaffold;
+
+import com.google.cloud.tools.jib.maven.TestPlugin;
+import com.google.cloud.tools.jib.maven.TestProject;
+import com.google.cloud.tools.jib.plugins.common.SkaffoldFilesOutput;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+/** Tests for {@link FilesMojoV2}. */
+public class FilesMojoV2Test {
+
+  @ClassRule public static final TestPlugin testPlugin = new TestPlugin();
+
+  @ClassRule
+  public static final TestProject simpleTestProject = new TestProject(testPlugin, "simple");
+
+  @ClassRule
+  public static final TestProject multiTestProject = new TestProject(testPlugin, "multi");
+
+  private static void verifyFiles(
+      Path projectRoot, String module, List<String> buildFiles, List<String> inputFiles)
+      throws VerificationException, IOException {
+
+    Verifier verifier = new Verifier(projectRoot.toString());
+    verifier.setAutoclean(false);
+    verifier.addCliOption("-q");
+    if (!Strings.isNullOrEmpty(module)) {
+      verifier.addCliOption("-pl");
+      verifier.addCliOption(module);
+      verifier.addCliOption("-am");
+    }
+    verifier.executeGoal("jib:" + FilesMojoV2.GOAL_NAME);
+
+    verifier.verifyErrorFreeLog();
+    Path logFile = Paths.get(verifier.getBasedir()).resolve(verifier.getLogFileName());
+    List<String> log = Files.readAllLines(logFile, StandardCharsets.UTF_8);
+
+    int begin = log.indexOf("BEGIN JIB JSON");
+    Assert.assertTrue(log.size() > begin + 2);
```

Comment:
nit: `assertThat(log.size(), ???.greaterThan(begin + 2))` to easily see the value of `log.size()` when this fails.

Reply:
Also, nit, but this test would pass even if `BEGIN_JIB_JSON` was not printed.